site/public/js/docker_interface.js

```diff
@@ -0,0 +1,63 @@
+
+function checkDockerInterfaceJobs(){
```

Comment:
use arrow functions for anonymous functions throughout this (e.g. `function (resolve, reject) {` -> `(resolve, reject) => {`